src/Components/DrillAnimationPage.js

```diff
@@ -25,6 +25,9 @@ const styles = StyleSheet.create({
   DrillAnimationPage: {
```

Comment:
Can you change this var name to be in camelCase as well, please ?

Reply:
And you can use destructuration as well for `styles`

Reply:
I'm not sure about styles destructuration

Discussing it there: https://github.com/UltimateApp/UltimateApp/pull/54#discussion_r403677912

Reply:
But the original comment about camelCase still stands :)